README.md

```
@@ -622,6 +622,27 @@ Your own `override def forkArgs = Seq(...)` replaces the list and drops the inje
+### Leiningen (`lein-plugin/`) [![Maven Central](https://img.shields.io/maven-metadata/v?metadataUrl=https%3A%2F%2Frepo1.maven.org%2Fmaven2%2Fnet%2Fexoego%2Fuika%2Flein-uika%2Fmaven-metadata.xml)](https://central.sonatype.com/artifact/net.exoego.uika/lein-uika)
+
+```clojure
+;; project.clj
+:plugins [[net.exoego.uika/lein-uika "VERSION_PLACEHOLDER"]]
+;; Optional: gate only on reachable violations, and suppress known false positives.
+:uika {:fail-on "reachable"
+       :exclude-files ["uika-exclude.toml"]}
+```
+
+```console
+$ lein uika dump-classpath                       # writes target/uika/classpath.json
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)